src/foam/nanos/logger/LogMessage.js

```diff
@@ -125,5 +125,13 @@ foam.CLASS({
     //   view: { class: 'foam.u2.view.PreView' },
     //   updateVisibility: 'RO'
     // }
+    {
+      documentation: 'Support searching by date range.  Created is a human readable date string',
+      name: 'lastModified',
```

Comment:
So this is the same as the 'created' field, just as a Date instead of a String? Should document this point so it isn't confusing to people.

Reply:
Um, that's what I thought said in the documentation.  I'll try to elaborate further.

Reply:
I see. Because this has a label: 'Created', I thought when it said "Created" in the docs it was talking about itself.
Having said: The "created" property is the same but in a human readable string format. would have been clearer to me.